### PR TITLE
150 wrong path   usersroles

### LINKED ISSUE #150: Wrong Path - Users&Roles
### Description - What happened? *

The path to the User&Roles submenu is not correct. The headline says "Users & Roles" (which is correct) but the path and also the name of the tab says "My Users". I have attached a screenshot.

### Expected Behavior *

?

### Observed Behavior *

?

### Steps to Reproduce

_No response_

### Context Information

_No response_

### Relevant log output

_No response_

### Screenshots

Wrong_path_UsersRoles

### PR DESCRIPTION
**Pull request by illfixit**
_What issues does this PR close?_
closes #150 Wrong Path - Users&Roles

image

```[tasklist]
### Checklist
- [ ] The PR title is short and expressive.
- [ ] I have updated the CHANGELOG.md. See [changelog_update.md](https://github.com/sovity/authority-portal/tree/main/docs/dev/changelog_updates.md) for more information.
- [ ] I have updated the Deployment Migration Notes Section in the CHANGELOG.md for any configuration / external API changes.
- [ ] I have performed a **self-review**
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -297,9 +298,9 @@ Major release, containing a UI rework and several new features.
 - Keycloak
   - Replace [MDS theme](authority-portal-keycloak/mds-theme) with the new version
   - Keycloak IAM needs to be upgraded to version 23.0.4
+
```
Comment (kamilczaja) [inline CHANGELOG.md:301]: Redundant blank line

**Review by kamilczaja — CHANGES_REQUESTED** [review]
image

There is a discrepancy between the tab name and the link name yet again.
It should be both either "Users & Roles" or both "Users and Roles".

But in no case "Users And Roles", we don't write connecting words with capitals.

I'm not sure which part of the code is responsible for that, so that's why it's in this comment.

File: CHANGELOG.md (hunk 1 above)
Comment (illfixit) [reply CHANGELOG.md]: no

**Review by kamilczaja — APPROVED** [review]
LGTM

**Pull request merged**